src/routes/index.js

```diff
@@ -8,5 +8,6 @@ router.get('/', function (req, res, next) {
 
 router.use('/posts', require('./posts'));
 router.use('/users', require('./users'));
+router.use('/donate', require('./donate'));
```

Comment:
@jiss02 복수형으르 쓰지 않는 이유가 있을까요~?
```suggestion
router.use('/donates', require('./donate'));
```

Reply:
동사라서용~!